lib/modules/apostrophe-pages/index.js

```diff
@@ -291,6 +291,76 @@ module.exports = {
       ], callback);
     };
 
+    // Task to add a page from apostrophe
+    // command line
+
+    self.apos.tasks.add('apostrophe-pages', 'add',
+      'Usage: node app apostrophe-pages:add title /slug type /parentslug\n\n' +
```

Comment:
A good idea to show an example here using quotation marks in the shell as will almost always be necessary with multiword titles